plugin/src/main/java/org/jenkinsci/plugins/structs/SymbolLookup.java

```diff
@@ -227,13 +228,22 @@ public static SymbolLookup get() {
             justification = "Jenkins.getInstance() can return null in theory.")
     @Nonnull public static Set<String> getSymbolValue(@Nonnull Class<?> c) {
         Set<String> symbolValues = new LinkedHashSet<String>();
-        if (Describable.class.isAssignableFrom(c) && !Descriptor.class.isAssignableFrom(c) && Jenkins.getInstance() != null) {
-            Descriptor d = Jenkins.getInstance().getDescriptor(c.asSubclass(Describable.class));
-            symbolValues.addAll(getSymbolValue(d));
+        Jenkins j = Jenkins.getInstanceOrNull();
+        if (Describable.class.isAssignableFrom(c) && !Descriptor.class.isAssignableFrom(c) && j != null) {
+            Descriptor<?> d = j.getDescriptor(c.asSubclass(Describable.class));
+            if (d != null) {
+                symbolValues.addAll(getSymbolValue(d));
+            }
         } else {
             Symbol s = c.getAnnotation(Symbol.class);
             if (s != null) {
                 Collections.addAll(symbolValues, s.value());
+            } else if (j != null && ParameterValue.class.isAssignableFrom(c)) { // TODO JENKINS-26093 hack, pending core change
+                try {
+                    symbolValues.addAll(getSymbolValue(c.getClassLoader().loadClass(c.getName().replaceFirst("Value$", "Definition"))));
```

Comment:
Would love if this could go away completely -- reflective yuckiness that hits the ClassLoader.   I know, I know, it's retained from previous logic but still, ew.

Reply:
Yeah, I have just not bothered to do the core change for that. In this PR, the logic is merely shuffled around a bit.